### ISSUE TITLE
Color contrast fixes for queue ID number and "Open" queue status badges

### ISSUE THREAD

**Issue by teshsopher** [tl-its-umich-edu/remote-office-hours-queue#207]
Currently, the blue badge (.badge-primary) for the Queue ID number and the green badge (.badge-success) for the "Open" status don't have enough color contrast. I recommend that both be darker to meet the 4.5 contrast ratio requirement.

**Comment by ssciolla** [comment]
@teshsopher, can you re-evaluate the green for the badge here? I changed a few things in the CSS in PR #201, and I think I resolved this.

I agree that we should align the blue in use here to what we have for the blue buttons throughout the U, to improve contrastI. I do wonder if the font-size and font-weight make it technically AA compliant...

**Comment by teshsopher** [comment]
The green works, thank you!

As far as the blue, it might be technically compliant. I know that the font-size says 90%, do you happen to know what that is in px or pt? If it's >18px or >14pt font and bold, then it would pass because large text only needs a contrast of 3:1

**Comment by ssciolla** [comment]
For this issue, we need to revisit the AA guidelines for color contrast and font size to make sure the badges are compliant. See https://www.w3.org/TR/WCAG21/. An accessibility checker may like Axe DevTools may be able to help check this, not sure.

Things may be fine as they are, but we need to check.

**Comment by zshahidumich** [comment]
Hey there! Nice to meet you all! My name is Zaid and I'm part of EECS 481 Software Engineering. I hope you're all doing well. I just wanted to drop a quick message here to make sure I've covered all my bases. I've already sent an email and joined the Google group, but I was wondering if there's a chance I could be assigned to this task. I am also dropping a message on issue #330. Thanks in advance!

**Comment by hemashahumich** [comment]
Here is what it looks like in grayscale:
![image](https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/155757888/76a6329d-21fc-40de-9d37-06544a7b5a41)

**Comment by hemashahumich** [comment]
I had sent an email to Kurt Mummer on 5/29 but created formal ticket todya 6/3 - https://teamdynamix.umich.edu/TDClient/30/Portal/Requests/TicketRequests/TicketDet?TicketID=wUyiVUMoCB8_

**Comment by hemashahumich** [comment]
We have received confirmation from the Accessibility team. so moving card to done column.

Detail on the request is below: 

[](https://teamdynamix.umich.edu/TDNext/Apps/31/Tickets/TicketDet.aspx?TicketID=6472458)

</html>Kurt Murmers updated ITS-Digital Accessibility Request, Ticket Number [6472458](https://teamdynamix.umich.edu/TDNext/Apps/31/Tickets/TicketDet.aspx?TicketID=6472458).

Details of your ticket:
Hi Hema,

Thank you so much for reaching out about this, and I apologize for missing your email. Yes, it is very helpful to open up a ticket with us for any questions like these!

I reviewed the updated color combinations and everything meets color contrast accessibility standards. 

Thank you for your work on this. I am going to close this ticket since everything looks good, but please feel free to re-open it if you have any additional questions or anything else we can help with!

Kurt
